test/common.js

```diff
@@ -14,6 +14,11 @@ exports.tmpDirName = 'tmp';
 exports.PORT = +process.env.NODE_COMMON_PORT || 12346;
 exports.isWindows = process.platform === 'win32';
 exports.isAix = process.platform === 'aix';
+exports.isLinuxPPCBE = (process.platform === 'linux') &&
```

Comment:
Nit that can totally be ignored: `isLinuxPpcBe` perhaps for consistency? (We don't do `isFreeBSD` or `isSunOS` for example.)